### PR TITLE
Created Dashboard Page using HTML and CSS and added multiple Javascript functions to it

### PR DESCRIPTION
**Pull request by Idiomet**
**Completed Tasks:**
- Created scripts for populating the semester and department
- Used fetch API to pull the courses for each department and semester combination
- Dynamically generates a table based on the amount of classes
- Clicking on the checkbox of that row will place that class under the Selected Classes section

**To Do:**
- Create mockup for the login page under the Notification section
- Complete the rest of the Notifications section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,101 +1,96 @@
...
-        >
-          <img src="assets/CSUN Logo.png" class="pr-2 pb-2 inline-flex" /><span
-            class=""
-            >Class Companion</span
-          ></a
-        >
+  <!-- Fonts -->
+  <link rel="preconnect"
+        href="https://fonts.gstatic.com">
+  <link rel="preconnect"
```
Comment (BrianLinggadjaja) [inline index.html]: Duplicate font call?

**Review by BrianLinggadjaja — CHANGES_REQUESTED** [review]
Looks awesome, just saw there might be a duplicate link at the head

File: index.html (hunk 1 above)
Comment (Idiomet) [reply index.html]: Removed the extra code.

**Comment by BrianLinggadjaja** [comment]
LGTM!

**Pull request merged**